snuba/datasets/processors/metrics_bucket_processor.py

```diff
@@ -96,9 +96,12 @@ def process_message(
             "project_id": message["project_id"],
             "metric_id": message["metric_id"],
             "use_case_id": message.get("use_case_id", "sessions"),
-            "timestamp": timestamp,
+            "timestamp": int(timestamp.replace(tzinfo=timezone.utc).timestamp()),
```

Comment:
@john-z-yang Should we revert this change? I thought Clickhouse expects a DateTime?

Reply:
Clickhouse client automatically converts datetime to ints, because that is how it is stored in the database itself
https://clickhouse.com/docs/en/integrations/python#write-formats
I prefer that we do not revert these changes because eventually we need to use the RustCompatProcessor and the tests would need to pass